scss/bitstyles/utilities/justify/justify.stories.mdx

```diff
@@ -1,45 +1,109 @@
 import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
 
-<Meta title="Utilities/justify-items" />
+<Meta title="Utilities/justify" />
 
-# Justify-items
+# Justify
 
 Flex and Grid share many alignment properties. These will have no effect on elements to which you haven’t given one of those classes (`.u-flex`, `.u-grid`).
 
-### Between
+Set `justify-content` to align flex children to the respecitve different edges on the cross-axis. This only affects layout when the combined elements are smaller than the available cross-axis:
```

Comment:
```suggestion
Set `justify` to align flex children to the respecitve different edges on the cross-axis. This only affects layout when the combined elements are smaller than the available cross-axis:
```

Same here: I don't see "content" being part of the class name 🔍 (but I may also just not get it).

Reply:
Thanks! The confusion is because of the difference in the classname and the name of the CSS property. I’ve made it clearer which one is meant for both of these 👍